documentation/release-latest/docs/rules/standard.md

```diff
@@ -59,22 +59,22 @@ Requires a blank line before any class or function declaration. No blank line is
 === "[:material-heart:](#) Ktlint"
 
     ```kotlin
-    const val FOO_1 = "foo1"
```

Comment:
`FOO_1` is used intentionally. Constant value which are deeply immutable property should be written in screaming case according to `property-naming` rule. Of course it should have been consistent with the disallowed example.